tests/Behat/Page/Admin/OrderCreatePage.php

```diff
@@ -12,31 +12,39 @@
 {
     public function addProduct(string $productName): void
     {
-        $this->getDocument()->selectFieldOption('Products', $productName);
-        $this->getDocument()->pressButton('Add product');
+        $this->getDocument()->clickLink('Add');
+        $this->getDocument()->selectFieldOption('Variant', $productName);
     }
 
     public function specifyShippingAddress(AddressInterface $address): void
     {
         $this->fillAddressData(
-            $this->getDocument()->find('css', '#shipping-address'),
+            $this->getDocument()->find('css', 'div[id*="shippingAddress"]'),
             $address
         );
     }
 
     public function selectShippingMethod(string $shippingMethodName): void
     {
-        $this->getDocument()->selectFieldOption('Shipping method', $shippingMethodName);
+        $shipmentsCollection = $this->getDocument()->find('css', '#sylius_admin_order_creation_new_order_shipments');
+
+        $shipmentsCollection->clickLink('Add');
+        $this->getSession()->wait(1000);
```

Comment:
Hmm, can we use `wait` here, but with some checks whether selecting that field option is possible?